lib/sync/queries.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
                   name
                 }
                 authoredDate
-                history(first: $per_page) {
+                history(first: 50) {
```

Comment:
Minor: how about pulling this value out into a variable so it's more clear what it represents?

```suggestion
                history(first: numberOfCommitsPerBranch) {
```

Reply:
👋 Good idea! Not sure how to do this without passing a variable into the query which would make this into a bigger change because we'd have to pass this variable in every spot we make this query. I think we can keep this hardcoded for now (I've seen dotcom queries like this as well) and then revisit the issue in the follow-up PR when debugging the page size retries?